### PR TITLE
Update UG About Section

### PR DESCRIPTION
**Pull request by mrmrinal**
Updated UG section 3 with the following
3.2 Application Home Window Breakdown
3.3 (partly) with explanation for Client Window and Contat & Policy Tab

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
 
 ### 3.2 Application Home Window Breakdown
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
```
Comment (bharathcs) [inline docs/UserGuide.md:95]: The alt text (in between `[]`) should be normal text, like Main Window Screenshot. No need for PascalCase :)

File: docs/UserGuide.md [hunk 2]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+In this section, you will be able to view a list of all the clients that have their records stored on DonnaFin. The index
+of a client corresponds to the index needed to be inputted in the `view` command. For example: if the client list is as
+per the image above, inputting the command `view 2` would take you to the client view of Bernice Yu, as she has the index
+of 2.
+
+The following attributes of your client is also viewable from the Client List Window:
+1. `Name`
+2. `Phone`
+3. `Address`
+4. `Email Address`
```
Comment (bharathcs) [inline docs/UserGuide.md:118]: Shouldn't be in code fences, as these are normal english words

File: docs/UserGuide.md (hunk 1 above)
Comment (bharathcs) [inline docs/UserGuide.md:95]: The alt text (in between []) should be normal text, like Main Window Screenshot. No need for PascalCase :)

File: docs/UserGuide.md [hunk 3]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the File button that allows the you to quickly exit the app, and the help 
+  button that redirects you to the User Guide for further guidance.
+
+#### 3.2.2 Command Box
+* The command box is where you can enter your commands in.
```
Comment (bharathcs) [inline docs/UserGuide.md:102]: Add a line that you can hit enter to execute it.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+![ClientView](images/ClientView.png)
+
+#### 3.3.1 Client Tab Buttons
+* These buttons are specific to the Client Window, and you can use these buttons to toggle through the different tabs
+that have been implemented to access and edit client information. The tabs are as follows:
+  1. `Contact`
+  2. `Policies`
+  3. `Assets`
+  4. `Liabilities`
+  5. `Notes`
```
Comment (bharathcs) [inline docs/UserGuide.md:143]: Again, same as before, ti desont need to be in code as these are normal concepts meant for ppl to read as if its typical English.

File: docs/UserGuide.md [hunk 5]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+#### 3.3.2 Contact Tab
+The `Contact Tab` is the Client View tab you first see when transitioning from the Home View to the Client View.
+Alternatively, this tab can also be accessed by pressing on the `Contact` button when in any of the Client View tabs,
+or through typing the command `tab c`, `tab contact` or `tab contacts`. You can access the following contact details of
+your client:
+1. `Name`
+2. `Phone`
+3. `Email`
+4. `Address`
+
```
Comment (bharathcs) [inline docs/UserGuide.md:156]: same as before

File: docs/UserGuide.md [hunk 6]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+This information is represented as shown in the picture below.
+
+![ClientContactTab](images/ClientContactTab.png)
+
+The Contact tab also supports the `edit` that allows you to change any of your client's contact details as
+shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#432-edit-clients-contact-information-edit)
+
+### 3.3.3 Policies Tab
+The `Policies Tab` is the Client View tab that stores all of your client's policies. This tab can be access by pressing on
```
Comment (DonnyS57) [inline docs/UserGuide.md:165]: *can be accessed

File: docs/UserGuide.md [hunk 7]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+the `Policies button` when in any of the Client View tabs, or through typing the command `tab p`, `tab policy` or `tab policies`.
+
+You will know that you are in the Policy tab if you see the **Policies** header on below the Result Display. You can access the total
```
Comment (DonnyS57) [inline docs/UserGuide.md:168]: *header below?

File: docs/UserGuide.md [hunk 8]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+commission that you have earned through looking at the number to the right of the **Total Policy Commisions**.
+
+The Policies are displayed in a Table View where each row represents a new policy, while each column represents a Field of
```
Comment (bharathcs) [inline docs/UserGuide.md:171]: if Table View is meant to be component, you should put it in code fences and stuff. Else, just use 'table' in lower case.

File: docs/UserGuide.md [hunk 9]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+
+The Contact tab also supports the `edit` that allows you to change any of your client's contact details as
+shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#432-edit-clients-contact-information-edit)
+
+### 3.3.3 Policies Tab
+The `Policies Tab` is the Client View tab that stores all of your client's policies. This tab can be access by pressing on
+the `Policies button` when in any of the Client View tabs, or through typing the command `tab p`, `tab policy` or `tab policies`.
+
+You will know that you are in the Policy tab if you see the **Policies** header on below the Result Display. You can access the total
+commission that you have earned through looking at the number to the right of the **Total Policy Commisions**.
```
Comment (DonnyS57) [inline docs/UserGuide.md:169]: through -> by ? (feels easier to read)

File: docs/UserGuide.md [hunk 10]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![MainWindowScreenshot.png](./images/MainWindowScreenshot.png)
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the File button that allows the you to quickly exit the app, and the help
```
Comment (bharathcs) [inline docs/UserGuide.md:98]: wrap File in quotes to indicate its a literal quotation from the app pls

File: docs/UserGuide.md [hunk 11]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+  1. `Contact`
+  2. `Policies`
+  3. `Assets`
+  4. `Liabilities`
+  5. `Notes`
+    
+The tabs can also be toggled through using the `tab` command as shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#431-switching-tabs-in-client-window-tab)
+
+#### 3.3.2 Contact Tab
+The `Contact Tab` is the Client View tab you first see when transitioning from the Home View to the Client View.
```
Comment (bharathcs) [inline docs/UserGuide.md:148]: it should not be in code fences here tbh, like all other uses of home view, client view, this is just a name for a component, not a class name or object or CLI usage etc.

**Review by bharathcs — CHANGES_REQUESTED** [review]
Hey mrinal, thanks for the UG work! Some formatting points to note!

File: docs/UserGuide.md (hunk 9 above)
Comment (bharathcs) [reply docs/UserGuide.md:169]: yep, by makes more sense! nice

File: docs/UserGuide.md [hunk 12]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the File button that allows the you to quickly exit the app, and the help 
+  button that redirects you to the User Guide for further guidance.
+
+#### 3.2.2 Command Box
+* The command box is where you can enter your commands in.
+
+#### 3.2.3 Results Display
```
Comment (Bluntsord) [inline docs/UserGuide.md:104]: Think we left it everywhere as result display right. Not results

File: docs/UserGuide.md (hunk 6 above)
Comment (bharathcs) [reply docs/UserGuide.md:165]: how about 'could be accessed'

File: docs/UserGuide.md [hunk 13]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+* The results display is where the action taken by DonnaFin in response to the command you keyed in is displayed.
```
Comment (Bluntsord) [inline docs/UserGuide.md:105]: The result display is where the response to the command entered is displayed.? More concise

File: docs/UserGuide.md [hunk 14]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+2. `Insurer`
+3. `Insured Value`
+4. `Yearly Premium`
+5. `Commission`
+
+Each policy has an index which is sorted by Ascending order, and if you would like to remove an existing policy, you can
+keep track of the index by looking at the number to the left of the policy name you would like to delete.
+
+
+The Contact tab also supports the `append` and `remove` command that allows you to add a new policy or
```
Comment (DonnyS57) [inline docs/UserGuide.md:187]: Contact -> Policy?

File: docs/UserGuide.md [hunk 15]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+
+### 3.3 Application Client Window Breakdown
+The Client window is accessed when you use the `view` command correctly. Though the client window shows content different
+from the main window, the `Quick Access Button`, `Command Box` and `Results Display` are all the same as explained 
+earlier in section 3.2. 
+
+<div markdown="block" class="alert alert-warning">:exclamation: **Caution:**
+
+Do take note that the `Command Box` is tab specific in the Client Window 
+so do take not that you are inputting the correct command in the correct tab.
```
Comment (Bluntsord) [inline docs/UserGuide.md:128]: Take note* typo here. Can be rephrased as well. please ensure you are inputting .....

File: docs/UserGuide.md [hunk 16]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+4. `Address`
+
+This information is represented as shown in the picture below.
+
+![ClientContactTab](images/ClientContactTab.png)
+
+The Contact tab also supports the `edit` that allows you to change any of your client's contact details as
+shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#432-edit-clients-contact-information-edit)
+
+### 3.3.3 Policies Tab
```
Comment (DonnyS57) [inline docs/UserGuide.md:164]: Should be ####

**Review by bharathcs — APPROVED** [review]

File: docs/UserGuide.md [hunk 17]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
 
-### 3.3 Client Explanation
+Here is an overview of the different elements that makes up the Home Window of DonnaFin
 
-### 3.4 Application Client Window Breakdown
+![Main Window.png](./images/MainWindowScreenshot.png)
 
-### 3.5 Command Format
+#### 3.2.1 Quick Access Buttons
+* This area gives you access to the 'File' button that allows the you to quickly exit the app, and the help
```
Comment (DonnyS57) [inline docs/UserGuide.md:98]: should be backtick not quotation

File: docs/UserGuide.md [hunk 18]
```diff
@@ -90,11 +90,106 @@ This marks the end of the quick start. Click
...
+This information is represented as shown in the picture below.
+
+![ClientContactTab](images/ClientContactTab.png)
+
+The Contact tab also supports the `edit` that allows you to change any of your client's contact details as
+shown [here.](https://ay2122s1-cs2103t-w16-1.github.io/tp/UserGuide.html#432-edit-clients-contact-information-edit)
+
+#### 3.3.3 Policies Tab
+The `Policies Tab` is the Client View tab that stores all of your client's policies. This tab could be accessed by pressing on
+the `Policies button` when in any of the Client View tabs, or by typing the command `tab p`, `tab policy` or `tab policies`.
```
Comment (DonnyS57) [inline docs/UserGuide.md:166]: the button should not be in the backtick (for consistency with the contact)

**Pull request merged**